CMakeLists.txt

```diff
@@ -138,6 +139,14 @@ IF (NOT HAVE_SYS_LDNS_H OR NOT LDNS_LIB)
     MESSAGE(FATAL_ERROR "ldns/ldns.h not found - libdns installed?")
 ENDIF()
 
+CHECK_INCLUDE_FILE(jansson.h HAVE_SYS_JANSSON_H)
```

Comment:
> On 15 Jun 2016, at 11:15, Kristian A. Hiorth <notifications@github.com> wrote:
> 
> In CMakeLists.txt <https://github.com/NEAT-project/neat/pull/90#discussion_r67126224>:
> 
> > @@ -138,6 +139,14 @@ IF (NOT HAVE_SYS_LDNS_H OR NOT LDNS_LIB)
> >      MESSAGE(FATAL_ERROR "ldns/ldns.h not found - libdns installed?")
> >  ENDIF()
> >  
> > +CHECK_INCLUDE_FILE(jansson.h HAVE_SYS_JANSSON_H)
> This check for libjansson has been duplicated somehow. Merge oops?
> 
> 
Yes. Will remove.

> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/NEAT-project/neat/pull/90/files/24b0d23d9732e1813e296d2d221beb851bec481a#r67126224>, or mute the thread <https://github.com/notifications/unsubscribe/ACejY-_Q0MRpgfQz5eDeB4M-SNBEIFxCks5qL8KbgaJpZM4Izozz>.
>